src/services/user.service.js

```diff
@@ -37,3 +38,59 @@ export const createUser = async (req, res) => {
 	}
 };
 
+const updateUser = (user, userInfo) => users.update(userInfo, {
+    where: user,
+    returning: true
+  });
+
+export const updateRole = (req, res, next) => {
+    const email = req.body.email;
+    const role = req.body.role;
+
+    	users.findOne({ where: { email: email } }).then(user => {
+        console.log(user)
+        if(user === null){
+            return res.status(404).json({ message: 'User not found' });
+        }
+
+        if (user.role === 'super_admin') {
```

Comment:
We are keeping `super admin` in the database, so I don't thing it would match that value of `super_admin`

Reply:
Done.